core-primitives/top-pool-author/src/author_tests.rs

```diff
@@ -152,35 +149,3 @@ fn create_author_with_filter<F: Filter<Value = TrustedOperation>>(
 		encryption_key,
 	)
 }
-
-fn trusted_call_signed() -> TrustedCallSigned {
-	let account = ed25519::Pair::from_seed(&TEST_SEED);
-	let call =
-		TrustedCall::balance_shield(account.public().into(), account.public().into(), 12u128);
-	call.sign(&KeyPair::Ed25519(account), 0, &mr_enclave(), &shard_id())
-}
-
-fn trusted_getter_signed() -> Getter {
-	let account = ed25519::Pair::from_seed(&TEST_SEED);
-	let getter = TrustedGetter::free_balance(account.public().into());
-	Getter::trusted(getter.sign(&KeyPair::Ed25519(account)))
-}
-
-fn create_indirect_trusted_operation() -> TrustedOperation {
-	let account = ed25519::Pair::from_seed(&TEST_SEED);
-	let trusted_call_signed = TrustedCall::balance_transfer(
-		AccountKeyring::Alice.public().into(),
-		AccountKeyring::Bob.public().into(),
-		1000u128,
-	)
-	.sign(&KeyPair::Ed25519(account), 1, &mr_enclave(), &shard_id());
-	TrustedOperation::indirect_call(trusted_call_signed)
-}
-
-fn mr_enclave() -> [u8; 32] {
-	[1u8; 32]
-}
-
-fn shard_id() -> ShardIdentifier {
-	BlakeTwo256::hash(vec![1u8, 2u8, 3u8].as_slice().encode().as_slice())
-}
```

Comment:
Moved these functions to a separate `test_fixtures.rs` file, so they can be reused.